libsnark/zk_proof_systems/plonk/prover.tcc

```diff
@@ -1028,27 +1017,28 @@ plonk_proof<ppT> plonk_prover<ppT>::compute_proof(
 
     printf("[%s:%d] Prover Round 2...\n", __FILE__, __LINE__);
     // - beta, gamma: permutation challenges - hashes of transcript of round 1
-    const libff::Fr<ppT> beta = transcript_hash.beta;
-    const libff::Fr<ppT> gamma = transcript_hash.gamma;
+    const libff::Fr<ppT> beta = hasher.get_hash();
```

Comment:
This value will be deterministic, won't it?
Should we not hash the first elements sent to the verifier?

Similarly below.  I cannot see any calls to `add_element` on the hasher.